<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ViewLink demo</string>
  </property>
  <widget class="QWidget" name="widgetDevStatus" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>760</y>
     <width>1280</width>
     <height>35</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Yaw:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbYaw">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Pitch:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbPitch">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Target Lat:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbTargetLat">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Target Lng:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbTargetLng">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Target Alt:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbTargetAlt">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Zoom:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbZoom">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Laser distance:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbLaserDistance">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetDevControl" native="true">
   <property name="geometry">
    <rect>
     <x>1009</x>
     <y>10</y>
     <width>261</width>
     <height>751</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>10000</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="PTZ_control">
      <property name="title">
       <string>PTZ Control</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="XSlider" name="SliderMoveSpeed">
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>angle:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="angle_input">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="angle_enter">
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="GaugeCloud" name="gaugeCloud"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Picture_control">
      <property name="title">
       <string>Picture Control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1" colspan="2">
        <widget class="QPushButton" name="btnStopRecord">
         <property name="text">
          <string>Stop record</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Image sensor:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnZoomIn">
         <property name="text">
          <string>ZoomIn</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btnZoomOut">
         <property name="text">
          <string>ZoomOut</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>IR color:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="cmbIRColor"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cmbImageSensor"/>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnStartRecord">
         <property name="text">
          <string>Start record</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnGimbalTakePhoto">
         <property name="text">
          <string>take photo</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="checkBoxPIP">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>PIP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Zoom Speed:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="XSlider" name="SliderMoveZoom">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="track_control">
      <property name="title">
       <string> Track Control</string>
      </property>
      <widget class="QPushButton" name="select_Oblect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Object </string>
       </property>
      </widget>
      <widget class="QPushButton" name="close_tracking">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Close Tracking</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="VLKVideoWidget" name="widgetMainVideo" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="connect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>960</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Connect</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>108</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Network Video:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>USB Camera:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>30</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TCP Gimbal:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,0" rowminimumheight="0,0">
     <item row="0" column="0">
      <widget class="QComboBox" name="cmbNetworkVideoURL">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnOpenNetworkVideo">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cmbUSBCamera"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnOpenUSBVideo">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>31</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditPort">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="cmbIPAddr">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnConnectTCPBroad">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnConnectTCP">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>吊舱连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>961</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GaugeCloud</class>
   <extends>QWidget</extends>
   <header>gaugecloud.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>xslider.h</header>
  </customwidget>
  <customwidget>
   <class>VLKVideoWidget</class>
   <extends>QWidget</extends>
   <header>VLKVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
